File: week-2/day-7-OOP/algo-caesar-cipher/js/test_cs.js
// Rewrite this in Unit Test

var cs = require("./caesarCipher");

test("Caesar Cipher with negative shift", () => {
  const result = caesarCipher("Boy! What a string!", -5);
  expect(result).toBe("Wjt! Rcvo v nomdib");
});

test("Caesar Cipher with positive shift", () => {
  const result = caesarCipher("Hello zach168! Yes here.", 5);
  expect(result).toBe("Mjqqt efhm168! Djx mjwj");
});

test("Caesar Cipher with negative shift and mixed case", () => {
  const result = caesarCipher("Hello Zach168! Yes here.", -5);
  expect(result).toBe("Czggj Uvxc168! Tzn czmz");
});

File: week-2/day-7-OOP/algo-caesar-cipher/python/test_cs.py
# Rewrite this in Unit Test

from caesar_cipher import caesar_cipher

def test_caesar_cipher():
    assert caesar_cipher("Boy! What a string!", -5) == "Wjt! Rcvo v nomdib!"
    assert caesar_cipher("Hello zach168! Yes here.", 5) == "Mjqqt efhm168! Djx mjwj."
    assert caesar_cipher("Hello Zach168! Yes here.", -5) == "Czggj Uvxc168! Tzn czmz."
